### DISCUSSION TITLE
Is assembly graph organelle sufficient?

### DISCUSSION THREAD

**Discussion by karthickbrlab** [Kinggerm/GetOrganelle#321] [Q&A]
after running getorganelle, i visualized using bandage.

Kindly check the attached graph. Does this an complete assembly.  WHETHER IT IS ORGANELLE SUFFICIENT

extended_K105.assembly_graph.fastg.extend-embplant_pt-embplant_mt.fastg

![graph](https://github.com/Kinggerm/GetOrganelle/assets/140624883/a1ad16c9-7ff1-4bc4-bb4b-f2c39329dcd0)

Secondly, there were single Inverted repeat sequence in the genome. Does the assembly merged two inverted repeat into one?

**Comment by JianjunJin** [comment]
1. It is organelle sufficient. You should obtain two fasta files out from `get_organelle_from_reads.py` (see [here](https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-are-the-differences-of-complete1path_sequencefasta-and-complete2path_sequencefasta-output-files) for why there are two).
2. This is a graph representation of the assembly, where two inverted repeats are identical and surely represented as one. However, the two repeats will be preserved in the linearized form - fasta files, see the same link above and also see the examples to learn more.

**Comment by karthickbrlab** [comment]
I just used getorganelle from assembly graph to extract the circular cp
genome from the fastg/gfa files..

I did visualized using ogdraw.. I think I have got it right.

Thanks for your assistance.
Karthick
